cogtiler/classify/aspect.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+def classify(data):
+    rgb = np.empty((3, data.shape[1], data.shape[2]), np.uint8)
```

Comment:
Can you document the shape of the data and the actions being performed?

Reply:
Sure - it's probably throw-away code though...